server/utils/validation/index.js

```diff
@@ -6,3 +6,5 @@ module.exports.appointmentDataValidation = require('./appointmentDataValidation'
 module.exports.historyLogSchema = require('./historyLogValidation');
 module.exports.appointmentIdValidation = require('./appointmentIdValidation');
 module.exports.appointmentDateTimeValidation = require('./appointmentDateTimeValidation');
+module.exports.appointmentStatusValidation = require('./appointmentStatusValidation');
```

Comment:
can you edit line 3 ?
```js
module.exports.appointmentDateValidation = require('./appointmentDateValidation');
```
and the module export itself

Reply:
ok